### PR TITLE
Ch 4

### PR DESCRIPTION
**Pull request by vtotient**
Separated the two tickets as requested. The functions.php file contains the get_template_directory. Created template-parts folder and title.php file. The custom field can be enabled/disabled with a check box by the client in the admin page. The index.php includes get_template_part to call the title.php. The wp generated code is also in the inc file titled title-cf.php, let me know if this name needs to change.

### CODE REVIEW

File: template-parts/title.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?php if( get_field('enable_title') && get_field('page_title') ): ?>
+  <div class='row pt-5'>
```
Comment (pritpalc) [inline template-parts/title.php]: Let's use double quotes when assigning values to attributes.

File: template-parts/title.php [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    <div class='col'>
+    <h1 class='font-weight-bold pl-5'><?php the_field('page_title'); ?></h1>
```
Comment (pritpalc) [inline template-parts/title.php]: Make sure the `h1` is properly indented.

File: functions.php [hunk 3]
```diff
@@ -7,4 +7,5 @@
 
 require get_template_directory() . '/inc/cleanup.php';
 require get_template_directory() . '/inc/enqueue.php';
-require get_template_directory() . '/inc/theme-support.php';
\ No newline at end of file
+require get_template_directory() . '/inc/theme-support.php';
+require get_template_directory() . '/inc/paragraph-cf.php';
```
Comment (pritpalc) [inline functions.php]: Did you mean `/inc/title-cf.php`?

File: index.php [hunk 4]
```diff
@@ -1,3 +1,7 @@
 <?php get_header(); ?>
   <h1>This is my index.</h1>
 <?php get_footer(); ?>
+
+<div class='container'>
```
Comment (vinielk) [inline index.php]: Move this container above the footer, otherwise it will appear below the footer.

**Comment by vinielk** [comment]
Don't commit the .DS_Store files to the repo, they are not needed. If possible can you add .DS_Store to the gitignore file. Let me know if you need any help.

**Pull request closed without merging**